cluster.yml

```diff
@@ -70,6 +70,7 @@
     - { role: kargo-defaults}
     - { role: kubernetes/master, tags: master }
     - { role: kubernetes-apps/network_plugin, tags: network }
+    - { role: kubernetes-apps/policy_controller, tags: policy-controller }
```

Comment:
Also, we may have to make some more complex condition in cluster.yaml, since Canal requires policy controller. So I'd rather kept those conditionals in role meta, not in cluster.yaml